components/CloudinaryImage.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Image, Transformation } from 'cloudinary-react';
+import path from 'path';
+
+class CloudinaryImage extends React.Component {
+  render() {
+    const { url, alt, maxWidth, aspectRatio } = this.props;
+
+    return (
+      <Image publicId={path.basename(url)} cloud_name="texas-justice-initiative" secure="true" alt={alt}>
+        <Transformation width={maxWidth} crop="scale" aspect_ratio={aspectRatio} />
```

Comment:
I called this prop `maxWidth` to communicate that it doesn't actually control the `width` attribute on the compiled `<img>` tag. Instead, it's used to specify the width of the image we'd like Cloudinary to return, and we want that to be largest width at which we'll display the image.

Does that seem intuitive, or can you think of a better name or way to handle this?

Reply:
I think it's fine. We could add a comment in to explain it, but I honestly think it's intuitive enough what we are trying to pass here.